unstructured/partition/lang.py

```diff
@@ -146,8 +146,20 @@ def prepare_languages_for_tesseract(languages: Optional[List[str]] = ["eng"]):
     if languages is None:
         raise ValueError("`languages` can not be `None`")
     converted_languages = list(
-        filter(None, [convert_language_to_tesseract(lang) for lang in languages]),
+        filter(
```

Comment:
Just a thought, but would we want to log a warning for any changes made to the languages users pass in (so the removal of duplicates is also logged in the warning message)?

Reply:
hmm i don't think thats necessary. In my opinion its similar to convert languages parameter to standard language code and we don't need to tell them the converted language code.

Reply:
cc@shreyanid ^^